### DISCUSSION TITLE
执行<leader>w 提示 E492

### DISCUSSION THREAD

**Discussion by qinjianwei** [ayamir/nvimdots#784] [General]
![image](https://github.com/ayamir/nvimdots/assets/17981557/91992d16-4128-4f0e-be41-e421b9eb3549)

**Comment by ayamir** [comment]
`Lazy sync` to make sure all of plugins installed.

**Comment by Jint-lzxy** [comment]
Closed due to inactivity. Please reactivate if further support is needed.
